packages/react-server-gulp-module-tagger/index.js

```diff
@@ -6,13 +6,19 @@ var replace = require("gulp-replace")
 //   - "__LOGGER__"
 //   - "__LOGGER__({ /* options */ })"
 var isWindows = ('win32' === process.platform)
-,   REPLACE_TOKEN = /(?:__LOGGER__|__CHANNEL__|__CACHE__)(?:\(\s*(\{[\s\S]*?\})\s*\))?/g
+,   DEFAULT_REPLACE_TOKEN = /(?:__LOGGER__|__CHANNEL__|__CACHE__)(?:\(\s*(\{[\s\S]*?\})\s*\))?/g
 ,   THIS_MODULE   = isWindows
 	? /(?:[^\\]+\\node_modules\\)?react-server-gulp-module-tagger\\index\.js$/
 	: /(?:[^\/]+\/node_modules\/)?react-server-gulp-module-tagger\/index\.js$/
 
 module.exports = function(config) {
 	config || (config = {});
+	var REPLACE_TOKEN;
+	if (config.token) {
+		REPLACE_TOKEN =  new RegExp("(?:" + config.token + ")(?:\\(\\s*(\\{[\\s\\S]*?\\})\\s*\\))?", 'g');
```

Comment:
Don't like having the regexp machinery duplicated...

Maybe this could be in a function, and the default could be defined using it?

Like:

```javascript
DEFAULT_REPLACE_TOKEN = tokenToRegexp("__LOGGER__|__CHANNEL__|__CACHE__")
```